flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -1404,6 +1405,10 @@ void OmpStructureChecker::Enter(const parser::OmpClause::Schedule &x) {
   }
 }
 
+void OmpStructureChecker::Enter(const parser::OmpClause::ScheduleModifier &x) {
```

Comment:
Can you use CHECK_SIMPLE_CLAUSE instead of writing this function?

Reply:
Since it's no longer a clause, not really needed.